lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/kbv/api/service/VerifiableCredentialService.java

```diff
@@ -142,23 +160,47 @@ private Object[] convertBirthDates(List<BirthDate> birthDates) {
     }
 
     private Object[] calculateEvidence(KBVItem kbvItem) {
-
         Evidence evidence = new Evidence();
         evidence.setTxn(kbvItem.getAuthRefNo());
-
-        EventProbe eventProbe = new EventProbe();
-
-        if (VC_THIRD_PARTY_KBV_CHECK_PASS.equalsIgnoreCase(kbvItem.getStatus())) {
+        if (checkKbvFailureWithContraIndicator(kbvItem)) {
+            evidence.setVerificationScore(VC_FAIL_EVIDENCE_SCORE);
+            evidence.setCi(new ContraIndicator[] {ContraIndicator.V03});
+            logVcScore("fail");
+        } else if (VC_THIRD_PARTY_KBV_CHECK_PASS.equalsIgnoreCase(kbvItem.getStatus())) {
             evidence.setVerificationScore(VC_PASS_EVIDENCE_SCORE);
-            eventProbe.addDimensions(Map.of(METRIC_DIMENSION_KBV_VERIFICATION, "pass"));
-            LOGGER.info("kbv pass");
-
+            logVcScore("pass");
         } else {
             evidence.setVerificationScore(VC_FAIL_EVIDENCE_SCORE);
-            eventProbe.addDimensions(Map.of(METRIC_DIMENSION_KBV_VERIFICATION, "fail"));
-            LOGGER.info("kbv fail");
+            logVcScore("fail");
         }
 
         return new Map[] {objectMapper.convertValue(evidence, Map.class)};
     }
+
+    private boolean checkKbvFailureWithContraIndicator(KBVItem kbvItem) {
```

Comment:
rename this method to `hasMultipleIncorrectAnswers`

I think the methods below can be deleted and the a single conditional statement should suffice, e.g.:
`kbvItem.getQuestionAnswerResultSummary().getAnsweredIncorrect() > 1`
This also matches the acceptance criteria and the quantity of questions asked is irrelevant